contribs/gmf/apps/mobile/js/mobile.js

```diff
@@ -140,4 +140,4 @@ app.MobileController.prototype.rightNavIsVisible = function() {
 };
 
 
-app.module.controller('MobileController', app.MobileController);
+appModule.controller('MobileController', app.MobileController);
```

Comment:
To be the same as in ngeo and gmf :-)
see: https://github.com/camptocamp/ngeo/pull/448/files#diff-c72fa8a7df6c1c2a1cf6b46a1eaca607L14